Day-01/c/Kinamura/Day-01.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+
+#define file "../../../../prompts/01input.txt" 
+#define testfile "../../../../prompts/01test.txt"
```

Comment:
Optimally you choose a path that's not so arbitrary and outside this repository's file structure

Reply:
done